spec/features/inviting_accepted_submissions_feature_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe 'inviting accepted submissions' do
+  let(:setting) { FactoryGirl.build(:setting, available_spots: 2) }
+  let!(:user) { FactoryGirl.create(:user) }
+
+  before do
+    ActionMailer::Base.deliveries.clear
+    allow(Setting).to receive(:get).and_return(setting)
+  end
+
+  it 'sends one email to every accepted submission' do
+    accepted_submissions = FactoryGirl.create_list(
+      :submission, 2,
+      :with_rates,
+      confirmation_status: 'not_avaible',
+      rates_num: setting.required_rates_num,
+      rates_val: 1)
+
+    login_as(user, scope: :user)
+    visit submissions_results_path
+    click_link('Send')
+    click_link('Send')
```

Comment:
@czorek It's to ensure that everyone gets only one email even if the button is clicked more than once.
I can see where the confusion comes from, I will improve the code to make it more clear about what it does.